src/libcharon/plugins/vici/python/vici/protocol.py

```diff
@@ -63,7 +63,7 @@ class Packet(object):
     @classmethod
     def _named_request(cls, request_type, request, message=None):
         requestdata = request.encode("UTF-8")
-        payload = struct.pack("!BB", request_type, len(request)) + request
+        payload = struct.pack("!BB", request_type, len(request)) + requestdata
```

Comment:
I think `len(request)` should also be changed. But an easier fix would be to just change `requestdata` to `request`. As it was before 22f08609f1b6aabdc1177e0570c943842478de72, which introduced `requestdata` but without actually using it (not sure why @martinwilli introduced the new variable, the original `request` isn't used in the function anywhere so reassigning the result of `encode()` to it should be fine - he even did so in other places, maybe just an oversight).